### PR TITLE
chore: update node-sass, now that it works with node 8

### PR DESCRIPTION
**Pull request by lynnjepsen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "lolex": "2.1.3",
     "mkdirp": "^0.5.1",
     "mocha": "^5.0.0",
-    "node-sass": "^4.0.0",
+    "node-sass": "^4.7.2",
```
Comment (kfranqueiro) [inline package.json]: Update package-lock?

Also, is there a reason this is set at 4.7.2 rather than the latest it will match (4.8.2)?

Comment (patrickrodee) [reply package.json]: I updated package-lock. 4.7.2 is the most recent release of node-sass on NPM

Comment (kfranqueiro) [reply package.json]: Oh. Yeah just double-checked and 4.8.0 and 4.8.2 on github are marked as pre-release. 4.8.1 oddly isn't. Dunno what's up there.

Comment (patrickrodee) [reply package.json]: All good! Thanks man 👍

**Pull request merged**